plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -528,6 +521,15 @@ data Frame uni fun
 
 type Context uni fun = [Frame uni fun]
 
+toExMemory :: (Closed uni, uni `Everywhere` ExMemoryUsage) => CekValue uni fun -> ExMemory
+toExMemory = \case
+    VCon c      -> memoryUsage c
+    VDelay {}   -> 1
+    VLamAbs {}  -> 1
+    VBuiltin {} -> 1
```

Comment:
Nothing to do with this PR, but I'm wondering if the last three should be zero.  I think the only case where this will apply at the moment is for `ifThenElse`, but the cost of that is zero so it doesn't matter what size the inputs are.